- name : "/expr_1"
  signal_when:
    condition: "not published"
    timeout: 1.0
    safety_critical: True
    process_indices: [4]
    repeat_exec: True
  signal_lambdas:
  - expression: "lambda msg : msg.data == True"
    timeout: 0.1
    safety_critical: False
    when_published: False
    process_indices: [0,1]
    repeat_exec: False
  - expression: "lambda msg : msg.data == False"
    timeout: 5.0
    safety_critical: True
    when_published: False
    process_indices: [2,3]
    repeat_exec: False
  execute:
  - log:
      message: "expression 1 is True"
      level: "info"
  - log:
      message: "expression 1 is not False"
      level: "info"
  - log:
      message: "expression 1 is False"
      level: "warn"
  - log:
      message: "expression 1 is not True"
      level: "warn"
  - log:
      message: "expression 1 is not published"
      level: "warn"
  timeout: 0.1
  default_notifications: True
  include: True


- name : "/expr_2"
  signal_when:
    condition: "not published"
    timeout: 1.0
    safety_critical: True
    process_indices: [4]
    repeat_exec: True
  signal_lambdas:
  - expression: "lambda msg : msg.data == True"
    timeout: 0.1
    safety_critical: False
    when_published: False
    process_indices: [0,1]
    repeat_exec: False
  - expression: "lambda msg : msg.data == False"
    timeout: 5.0
    safety_critical: True
    when_published: False
    process_indices: [2,3]
    repeat_exec: False
  execute:
  - log:
      message: "expression 2 is True"
      level: "info"
  - log:
      message: "expression 2 is not False"
      level: "info"
  - log:
      message: "expression 2 is False"
      level: "warn"
  - log:
      message: "expression 2 is not True"
      level: "warn"
  - log:
      message: "expression 2 is not published"
      level: "warn"
  timeout: 0.1
  default_notifications: True
  include: False

